src/components/blocks/CardBody.tsx

```diff
@@ -0,0 +1,20 @@
+import { AttributesBlock } from 'types/AttributesBlock';
+import { UtkwdsCardBodyBlockAttributes } from 'client';
+import BlockRouter from 'components/BlockRouter';
+
+interface Props {
+  attributes: Partial<UtkwdsCardBodyBlockAttributes>;
+  innerBlocks?: AttributesBlock[];
+}
+
+const CardBody = ({
+  attributes: { className, textColor },
+  innerBlocks,
+}: Props) => (
+  <div className={`'card-body' ${className || ''} ${textColor || ''}`}>
```

Comment:
Need to remove the single-quote marks around `card-body`

Reply:
@MichaelAllenWarner I have removed the single-quote around card-body.